gradio/chat_interface.py

```diff
@@ -73,9 +73,9 @@ def __init__(
         analytics_enabled: bool | None = None,
         submit_btn: str | None | Button = "Submit",
         stop_btn: str | None | Button = "Stop",
-        retry_btn: str | None | Button = "🔄  Retry",
-        undo_btn: str | None | Button = "↩️ Undo",
-        clear_btn: str | None | Button = "🗑️  Clear",
+        retry_btn: str | None | Button = "Retry",
+        undo_btn: str | None | Button = "Undo",
+        clear_btn: str | None | Button = "Clear",
```

Comment:
I'm not a huge fan of removing the emojis. For two reasons: now the buttons look too monotonous to me, and we lose the benefits of the buttons being immediately clear and good for i18n. Why remove them?

Reply:
I don't love the emojis, but I would like us to keep some 'symbols' for the reasons you mentioned. I think some nice clean icons would be better.

Reply:
True now that we support icons in buttons, we could have some cleaner icons instead by default

Reply:
ok will revert but I think the emojis are ugly